website/content/docs/auth/saml.mdx

```diff
@@ -0,0 +1,158 @@
+within a [SAML V2.0](https://saml.xml.org/saml-specifications) Identity Provider.
+Authentication is suited for human users by requiring interaction with a web browser.
+
+## Authentication
+
+### Via the UI
+
+1. Select "SAML" from the "Method" select box.
+1. Enter a role name into the "Role" input field or leave blank to use
+   the [default role](/vault/api-docs/auth/saml#default_role).
+1. Press "Sign in with SAML Provider" and complete the authentication with the
+   configured SAML Identity Provider.
+
+### Via the CLI
+
+The CLI login defaults to the path of `/saml`. If this auth method was enabled at a
+different path, specify `-path=/my-path` in the CLI.
+
+```shell-session
+$ vault login -method=saml role=admin
+
+Complete the login via your SAML provider. Launching browser to:
+
+    https://company.okta.com/app/vault/abc123eb9xnIfzlaf697/sso/saml?SAMLRequest=fJI9b9swEIZ3%2FwqBu0SJ%2FpBDRAZce4iBtDViN0MX40Sda...
+```
+
+The default browser will open to the generated URL where the user must authenticate with
+the configured SAML Identity Provider. The URL may be manually entered into the browser
+if it cannot be automatically opened.
+
+The CLI login behavior may be customized with the following optional parameters:
+
+- `skip_browser` (default: `false`): If set to `true`, automatic launching of the default
+  browser will be skipped. The SAML Identity Provider URL must be manually entered in a
+  browser to complete the authentication flow.
+- `abort_on_error` (default: `false`): If set to `true`, failure to launch the default
+  browser will result in a non-zero exit of the CLI and an error displayed.
+
+## Configuration
+
+Auth methods must be configured in advance before users or machines can
+authenticate. These steps are usually completed by an operator or configuration
+management tool.
+
+1. Enable the SAML auth method:
+
+   ```shell-session
+   $ vault auth enable saml
+   ```
+
+1. Use the `/config` endpoint to save the configuration of your SAML Identity Provider and
+   set the default role. You can configure the trust relationship with the SAML Identity
+   Provider by either providing a URL for its Metadata document:
+
+   ```shell-session
+   $ vault write auth/saml/config \
+      default_role=admin \
+      idp_metadata_url=https://company.okta.com/app/abc123eb9xnIfzlaf697/sso/saml/metadata \
+      entity_id="https://my.vault/v1/auth/saml" \
+      acs_urls="https://my.vault/v1/auth/saml/callback"
+   ```
+
+   or by setting the configuration Metadata manually:
+
+   ```shell-session
+   $ vault write auth/saml/config \
+      default_role=admin \
+      idp_sso_url=https://company.okta.com/app/abc123eb9xnIfzlaf697/sso/saml \
+      idp_entity_id=https://www.okta.com/abc123eb9xnIfzlaf697 \
+      idp_cert=@path/to/cert.pem \
+      entity_id="https://my.vault/v1/auth/saml" \
+      acs_urls="https://my.vault/v1/auth/saml/callback"
+   ```
```

Comment:
````suggestion
1. Use the `write` command to save your SAML identity provider configuration to
   the desired path, for example `auth/saml/config \`. You can configure the
   trust relationship with the SAML identity provider in two ways. You can
   provide a URL for the relevant metadata document:

    ```shell-session
    $ vault write auth/saml/config                        \
        default_role=admin                                \
        entity_id="https://my.vault/v1/auth/saml"         \
        acs_urls="https://my.vault/v1/auth/saml/callback" \
        idp_metadata_url=https://company.okta.com/app/abc123eb9xnIfzlaf697/sso/saml/metadata
    ```
   
   Or, you can set the metadata details manually:

    ```shell-session
    $ vault write auth/saml/config                              \
        default_role=admin                                      \
        entity_id="https://my.vault/v1/auth/saml"               \
        acs_urls="https://my.vault/v1/auth/saml/callback"       \
        idp_cert=@path/to/cert.pem                              \
        idp_entity_id=https://www.okta.com/abc123eb9xnIfzlaf697 \
        idp_sso_url=https://company.okta.com/app/abc123eb9xnIfzlaf697/sso/saml
    ```
````

In general, we shouldn't refer to CLI commands as endpoints. Reserve "endpoint" for explicit API calls (e.g., cURL, SDK code) and refer to calls wrapped by the CLI as CLI commands.